config/v1/types_apiserver.go

```diff
@@ -63,6 +66,39 @@ type APIServerNamedServingCert struct {
 	ServingCertificate SecretNameReference `json:"servingCertificate"`
 }
 
+type APIServerEncryption struct {
+	// type defines what encryption type should be used to encrypt resources at the datastore layer.
+	// When this field is unset (i.e. when it is set to the empty string), identity is implied.
```

Comment:
`When this field is unset, the default behaviour is applied (which is "no encryption" today, but might change in future releases).`

Wouldn't say much more.

Reply:
I prefer my wording.  It's more direct and clearly spells out what we do today, what we want to do tomorrow, and what you can change to enforce a particular behavior.

Reply:
I think mine is more direct, and less verbose and does include promises.

Reply:
I don't care too much about most of this paragraph, only about "That is, encryption will be on by default in a future release" which IMO does not belong here. We have not decided anything and should certainly not promise undecided things. And even if we did, this should be about the semantics of the current API not some nebulous future release.